docs/source/guide/pt-4-low-level.md

```diff
@@ -0,0 +1,126 @@
+
+First let's define our circuit:
+```{code-cell} ipython3
+from cirq import LineQubit, Circuit, CZ, CNOT
+
+a, b, c, d  = LineQubit.range(4)
+circuit = Circuit(
+    CNOT.on(a, b),
+    CZ.on(b, c),
+    CNOT.on(c, d),
+)
+
+print(circuit)
+```
+Now, we can see what happens when we apply the PT functions, through {func}`.twirl_CNOT_gates()` and the subsequent {func}`.twirl_CZ_gates()`
+```{code-cell} ipython3
+from mitiq import pt
+
+circuit_to_twirl = circuit.copy()
+CNOT_twirled_circuits = pt.twirl_CNOT_gates(circuit_to_twirl, num_circuits=10)
+twirled_circuits = [
+    pt.twirl_CZ_gates(c, num_circuits=1)[0] for c in CNOT_twirled_circuits
+]
+print(CNOT_twirled_circuits[0:1])
+print(twirled_circuits[0:1])
+```
+We see that we return lists of the randomly twirled circuits, and so we must take a simple average over their expectation values.
+
+## Executing the modified circuits
+Now that we have our twirled circuits, let's simulate some noise and execute those circuits, using the {class}`mitiq.Executor` to collect the results.
+```{code-cell} ipython3
+from cirq import DensityMatrixSimulator, amplitude_damp
+from mitiq import Executor
+
+
+def execute(circuit, noise_level=0.003):
+    """Returns Tr[ρ |00..⟩⟨00..|] where ρ is the state prepared by the circuit
+    executed with depolarizing noise.
+    """
+    noisy_circuit = circuit.with_noise(amplitude_damp(noise_level))
+    rho = DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix
+    return rho[0, 0].real
+
+executor = Executor(execute)
+expvals = executor.evaluate(twirled_circuits, None)
+```
+
+
+## Estimate expectation value by averaging over randomized twirling circuits
+Pauli Twirling doesn't require running the circuit at different noise levels or with different noise models. It applies a randomized sequence of Pauli operations within the same quantum circuit and averages the results to reduce the effect of the noise.
+
+```{code-cell} ipython3
+import numpy as np
+from typing import cast
+
+average = cast(float, np.average(expvals))
+print(average)
+```
+
+Keep in mind, ths code is for illustration and that the noise level, type of noise (here amplitude damping), and the observable need to be adapted to the specific experiment.
+
+If executed on a noiseless backend, a given `circuit_with_pt` and `circuit` are equivalent.
+On a real backend, they have a different sensitivity to noise. The core idea of the PT technique is that,
+`circuits_with_pt` (hopefully) tailors the noise into stochastic Pauli channels, such that a simple average over results
+will return a mitigated result.
+
+As a final remark, we stress that the low-level procedure that we have shown is exactly what {func}`.execute_with_pt()` does behind the scenes.
+Let's verify this fact: 
+
+```{code-cell} ipython3
+np.isclose(
+  pt.execute_with_pt(circuit, executor),
+  average,
+)
+```
```

Comment:
Unfortunately, the results displayed don't match the text- `np.isclose` asserts `False`, see https://mitiq--1848.org.readthedocs.build/en/1848/guide/pt-4-low-level.html .